apps/web/app/(app)/environments/[environmentId]/integrations/google-sheets/components/ManageIntegration.tsx

```
@@ -45,7 +45,12 @@ export const ManageIntegration = ({
   const handleDeleteIntegration = async () => {
     try {
       setisDeleting(true);
-      await deleteIntegrationAction({ integrationId: googleSheetIntegration.id });
+      const deleteIntegrationActionResult = await deleteIntegrationAction({
+        integrationId: googleSheetIntegration.id,
+      });
+      if (deleteIntegrationActionResult && deleteIntegrationActionResult.serverError) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)